src/locale/locales/it/messages.po

```diff
@@ -6368,12 +5118,7 @@ msgstr "Impostazioni risposte"
 msgid "Reply settings are chosen by the author of the thread"
 msgstr "Le impostazioni delle risposte sono scelte dall'autore del thread"
 
-#~ msgctxt "description"
-#~ msgid "Reply to <0/>"
-#~ msgstr "In risposta a <0/>"
-
-#: src/view/com/post/Post.tsx:204
-#: src/view/com/posts/FeedItem.tsx:553
+#: src/view/com/post/Post.tsx:204 src/view/com/posts/FeedItem.tsx:553
 msgctxt "description"
 msgid "Reply to <0><1/></0>"
 msgstr "Rispondi a <0><1/></0>"
```

Comment:
Here should be "Risposta a <0><1/></0>"
![image](https://github.com/user-attachments/assets/e2c0434f-8935-4bc6-9506-89c2295d4e77)